DCGM builds for multiple CUDA versions, but our CUDA packages aren't
coinstallable.  To work around this, we change the CUDA discovery mechanism to
point at CUDA 13 for 11 and 12 and then remove the "incorrectly" built
artifacts afterwards.
--- a/cmake/FindCuda.cmake
+++ b/cmake/FindCuda.cmake
@@ -152,6 +154,16 @@ foreach(_Cuda_VERSION IN LISTS _Cuda_VERSIONS)
         "${Cuda_${_Cuda_MAJOR_VERSION}_cublasLt_LIBRARY}")
 endforeach()
 
+set(CUDA11_INCLUDE_DIR "${CUDA13_INCLUDE_DIR}")
+set(CUDA11_STATIC_LIBS "${CUDA13_STATIC_LIBS}")
+set(CUDA11_LIBS "${CUDA13_LIBS}")
+set(CUDA11_STATIC_CUBLAS_LIBS "${CUDA13_STATIC_CUBLAS_LIBS}")
+
+set(CUDA12_INCLUDE_DIR "${CUDA13_INCLUDE_DIR}")
+set(CUDA12_STATIC_LIBS "${CUDA13_STATIC_LIBS}")
+set(CUDA12_LIBS "${CUDA13_LIBS}")
+set(CUDA12_STATIC_CUBLAS_LIBS "${CUDA13_STATIC_CUBLAS_LIBS}")
+
 unset(_Cuda_LIBRARY)
 unset(_Cuda_FOUND)
 unset(_Cuda_VERSION)
